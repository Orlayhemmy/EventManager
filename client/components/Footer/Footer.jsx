/* eslint-disable  */
import React from 'react';

/**
 * @description Footer component
 */
export default class Footer extends React.Component {
  render() {
    return (
      <div className="col-lg-12 text-center footer pt-3">
        <p id="footer" className="footer-text">
          eCenter 2017
        </p>
      </div>
    );
  }
}
