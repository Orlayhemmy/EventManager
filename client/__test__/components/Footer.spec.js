import React from 'react';
import renderer from 'react-test-renderer';
import expect from 'expect';
import Footer from '../../components/Footer/Footer';

describe('Footer Component', () => {
  const wrapper = shallow(<Footer />);
  it('should render without throwing error', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('check the length of the containing div in the component', () => {
    expect(wrapper.find('#footer').length).toEqual(1);
  });
});
