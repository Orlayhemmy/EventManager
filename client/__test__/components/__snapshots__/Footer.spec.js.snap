// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should render without throwing error 1`] = `
<div
  id="footer"
>
  <footer>
    <div
      className="col-12 text-center footer"
    >
      <p>
        eCenter 2017
      </p>
    </div>
  </footer>
</div>
`;
